web/src/app/users/UserDetails.js

```diff
@@ -254,6 +262,13 @@ export default function UserDetails(props) {
           </Grid>
         }
       />
+      {showEdit && (
+        <UserEditDialog
+          onClose={() => setShowEdit(false)}
+          userID={props.userID}
+          role={user.role}
```

Comment:
To simplify the interface of this component, we have the ability to get the role of the user from the `useSessionInfo` call on L22 in `UserEditDialog.js`:

```
const { ready: isSessionReady, userID: currentUserID } = useSessionInfo()
```

->

```
const {
  isAdmin,
  ready: isSessionReady,
  userID: currentUserID
} = useSessionInfo()
```